pom.xml

```diff
@@ -192,11 +192,19 @@
       <artifactId>java-driver-metrics-micrometer</artifactId>
       <version>${driver.version}</version>
     </dependency>
+    <!--Note, using Java Driver QueryBuilder from a preview build. See https://github.com/stargate/data-api/issues/1509 -->
     <dependency>
       <groupId>com.datastax.oss</groupId>
       <artifactId>java-driver-query-builder</artifactId>
-      <version>${driver.version}</version>
-    </dependency>
+      <version>4.19.0-preview1</version>
```

Comment:
I think base version like 4.19.0 makes sense to use as the base of version id.
I know the version we use here does not really matter a lot (since we specify exact filename), but it would be nice to include more information for our own benefit, like date it was built. Plus that it is for use by data-api. So something like this (it's bit long but shows the idea)

     4.19.0.data-api-2024-10-24

?

Reply:
Also: probably more important, should include information on where the jar comes from. That is, which sources are used to build (link to repo, if public, branch).

Reply:
thanks, Tatu, fixed